walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js

```diff
@@ -56,6 +56,7 @@ function addQuoteToDom(quote) {
  * whichever syntax makes the most sense to you.
  */
 function getRandomQuoteUsingArrowFunctions() {
+  console.log('Fetching quote with arrow functions.')
```

Comment:
are these changes intended to go in this PR?

Reply:
No I am confused as to why these are showing up in this PR.

Reply:
These changes you guys are commenting on are from last week's project where I needed to change the Fetch a Quote button to a Get a Random Quote fact. These changes are not in any file on my branch so i'm not sure why these would be included.

Reply:
do you need to re-commit?